src/compiler/diagnosticMessages.json

```
@@ -3016,7 +3016,10 @@
         "category": "Error",
         "code": 2792
     },
-
+    "Type '{0}' can only be iterated through when using the '--downlevelIteration' flag.": {

```

This is the error message suggested by @DanielRosenwasser in #16022. I'm wondering if it should also mention a higher output target?

```suggestion
    "Type '{0}' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.": {

```

---

That seems better, though what if a user can't do either? What's the alternative to `Array.from`? `Array.prototype.slice.call`?

---

👍 or `[].slice.call` for short.